app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,24 @@
+module AdminHelper
+  def show_role(user)
+    if user.enable?
+      if user.user?
```

Comment:
I don't think so we need nesting of `if` condition here.

Reply:
Sir I have removed the else part of first if. So if a user is disabled there will be no action available for this. the first condition is needed for checking to status and second if is for checking role of the user. I have little confusion here.

Reply:
Thank @AmmadJavaid for explaining so well.